src/index.d.ts

```diff
@@ -18,17 +18,82 @@ import {Bucket} from '@google-cloud/storage';
 import * as _firestore from '@google-cloud/firestore';
 import {Agent} from 'http';
 
+/**
+ * `admin` is a global namespace from which all Firebase Admin
+ * services are accessed.
+ */
 declare namespace admin {
+
+  /**
+  * `FirebaseError` is a subclass of the standard JavaScript `Error` object. In
+  * addition to a message string and stack trace, it contains a string code.
+  */
   interface FirebaseError {
+    /**
+     * Error codes are strings using the following format: `"service/string-code"`.
+     * Some examples include `"auth/invalid-uid"` and
+     * `"messaging/invalid-recipient"`.
+     *
+     * While the message for a given error can change, the code will remain the same
+     * between backward-compatible versions of the Firebase SDK.
+     */
     code: string;
+    /**
```

Comment:
Lets add some spacing here, to separate it from the previous block. Here and elsewhere.